bestiary/core/db.py

```diff
@@ -328,6 +332,79 @@ def delete_project(trxl, project):
                        target=str(op_args['id']))
 
 
+def link_parent_project(trxl, project, parent_project):
+    """Link a project object with another with a child-parent relation.
+
+    This method sets the `parent_project` field value from `project` with the
```

Comment:
Maybe all the checks you did in `move_project` would work better here. Otherwise, this will mean you can have a graph with loops and not a tree.